src/OmniSharp.Roslyn/Services/AbstractCodeActionProvider.cs

```diff
@@ -54,9 +74,16 @@ protected AbstractCodeActionProvider(string providerName, ImmutableArray<Assembl
             }
             catch (Exception ex)
             {
-                throw new InvalidOperationException($"Failed to create instrance of {type.FullName} in {type.AssemblyQualifiedName}.", ex);
+                if (this._throwOnException)
+                {
+                    throw new InvalidOperationException($"Failed to create instrance of {type.FullName} in {type.AssemblyQualifiedName}.", ex);
+                }
+                else
+                {
+                    _logger.LogWarning($"Failed to create instrance of {type.FullName} in {type.AssemblyQualifiedName}.\n{ex.ToString()}");
```

Comment:
Could be either, one the one had we have many code actions that cannot be instantiated property because we don't support their inputs.  On the other it is a full exception.

I'm fine with switching it though.